tools/create_snapshot_artifact/main.py

```diff
@@ -24,7 +24,6 @@
 from framework.utils_cpuid import CpuVendor, get_cpu_vendor
 from host_tools.cargo_build import get_firecracker_binaries
 
-
```

Comment:
nit: accidental change?

Reply:
Nope, CI was failing due to this extra line. Separated change to its own commit for clarity